bootloader/wscript

```diff
@@ -661,6 +661,9 @@ def build(ctx):
     if ctx.env.CC_NAME == 'msvc':
         # Do not strip bootloaders when using MSVC.
         features = ''
+        
+        ctx.env['WINRC_TGT_F'] = '/fo'
+        ctx.env['WINRC_SRC_F'] = ''
```

Comment:
This to set the correct linker arguments when using MSVC. `WINRC_TGT_F` is used to determine the argument to use when linking objects and `-o` is the default (which MSVC doesn't understand). Let me see whether I can move this...